source/parser.hera

```diff
@@ -2898,6 +2898,12 @@ NamedImports
 FromClause
   From __ ModuleSpecifier
 
+# https://github.com/tc39/proposal-import-assertions
+# https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-5.html#import-assertions
+# NOTE: Forbid newline before `assert` which could mean a function call
+ImportAssertion
+  _? "assert" NonIdContinue __ ObjectLiteral
```

Comment:
Using `_*` instead of `__` should allow nested object literals to work here.

Reply:
I notice you use `_*` in a few places but `_?` seems sufficient, as `_` already has a `+` in it.

Reply:
`_?` is sufficient and probably leads to a shallower parse tree so maybe we should tend towards that.